plugins/github-release-manager/README.md

```diff
@@ -0,0 +1,49 @@
+# GitHub Release Manager (GRM)
+
+## Overview
+
+`GRM` enables developers to manage their releases without having to juggle git commands.
+
+Does it build and ship your code? **No**.
+
+What `GRM` does is manage your **[releases](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository)** on GitHub, building and shipping is entirely up to you as a developer to handle in your CI.
+
+`GRM` is built with industry standards in mind and the flow is as follows:
+
+![](./src/cards/info/flow.png)
+
+> **GitHub**: The source control system where releases reside in a practical sense. Read more about GitHub releases [here](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/. Note that this plugin works just as well with GitHub Enterprise)
```

Comment:
There's something wrong with the link and the parenthesis on this line; can you check?

Reply:
Addressed in https://github.com/backstage/backstage/pull/5299/commits/c003e06cc004cc085153112af6240a30e7d73200